core/storage/user/gae_models_test.py

```diff
@@ -1,4 +1,4 @@
-# coding: utf-8
+ # coding: utf-8
```

Comment:
Please remove the change in this line. Always check the "Files Changed" tab to ensure you didn't make any unexpected changes.

Reply:
Ack